test/evergreen/setup_spawn_host.sh

```diff
@@ -23,6 +23,21 @@ if [[ -n $WT_ARCHIVE ]]; then
     tar --wildcards -xzf $WT_ARCHIVE
 fi
 
+# Setup the gdb environment if there are core dumps present in the artefacts.
+GDB_CORE_DUMP=$(ls ${HOME}/wiredtiger/cmake_build/*.core | head -n 1 2>/dev/null)
+if [[ -n $GDB_CORE_DUMP ]]; then
+    # Read the wiredtiger shared library and find the old source directory using readelf and fix up
+    # the expected path to the correct path on the new machine.
+    LIB_PATH=$(ls ${HOME}/wiredtiger/cmake_build/*.so* | head -n 1 2>/dev/null)
+    OLD_WT_PATH=$(readelf --debug-dump=info ${LIB_PATH} | grep "wiredtiger" | head -n 1 | sed -e 's/wiredtiger.*/wiredtiger/' -e 's/[^\/]*//')
+    cat >> ~/.gdbinit << EOF
+set solib-search-path ${HOME}/wiredtiger/cmake_build/lang/python:${HOME}/wiredtiger/cmake_build:${HOME}/wiredtiger/TCMALLOC_LIB/lib
```

Comment:
I have only ever had to do `set solib-search-path ${HOME}/wiredtiger/cmake_build`, are all of the others necessary?

Reply:
yeah the main bulk of the gdb trace comes from the **${HOME}/wiredtiger/cmake_build**, the others are actually also quite necessary but are less important.